redaxo/src/addons/media_manager/lib/effects/effect_resize.php

```diff
@@ -20,22 +20,20 @@ public function __construct()
 <script type="text/javascript">
 <!--
 
-(function($) {
-    $(function() {
-        var $fx_resize_select_style = $("#media-manager-rex-effect-resize-style-select");
-        var $fx_resize_enlarge = $("#media-manager-rex-effect-resize-allow-enlarge-select").parent().parent();
-
-        $fx_resize_select_style.change(function(){
-            if(jQuery(this).val() == "exact")
-            {
-                $fx_resize_enlarge.hide();
-            }else
-            {
-                $fx_resize_enlarge.show();
-            }
-        }).change();
-    });
-})(jQuery);
+$(function() {
+    var $fx_resize_select_style = $("#media-manager-rex-effect-resize-style-select");
+    var $fx_resize_enlarge = $("#media-manager-rex-effect-resize-allow-enlarge-select").parent().parent();
```

Comment:
wäre es nicht sinnvoller mit dem `rex-js-` Prefix zu arbeiten?

Reply:
redest du von der variable oder dem selector?

Reply:
@staabm dem Selector. Nutzen wir teilweise bereits im Backend. Einfach auch um im HTML zu erkennen, dass hier JS ausgeführt wird und wenn der Selektor angepasst wird auch JS angepasst werden müsste..

Reply:
dieser selektor ist geniert und diesen jetzt zu ändern wird vermutlich anpassungen erfordern in effekten die andere ausserhalb des cores gebaut haben.

den können wir jetzt nicht mehr so einfach ändern meiner Meinung nach.
Erzeugt arbeit für andere ohne merklichen Mehrwert.